docs/pages/preview/custom-build-config-schema.mdx

```diff
@@ -0,0 +1,280 @@
+
+#### `steps.run`
+
+The `run` key is used to trigger a set of instructions. For example, a `run` key is used to install dependencies using the `npm install` command:
+
+```yaml
+build:
+  name: Install npm dependencies
+  steps:
+    # @info #
+    - run:
+        name: Install dependencies
+        command: npm install
+    # @end #
+```
+
+You can also `steps.run` to execute inline bash shell commands:
+
+```yaml
+build:
+  name: Run inline shell commands
+  steps:
+    # @info #
+    - run: echo "Hello world"
+    # @end #
+```
+
+#### `steps.name`
+
+The name of a step that is used in a `build` to identify which step triggers in the build logs.
+
+#### `steps.command`
+
+The `command` is defins a custom shell command to run when a step is executed. It is **required** to define a command for each step and can be a multiline bash shell command.
+
+#### `steps.working_directory`
+
+The `working_directory` is used to define an existing directory from the project's root directory. For example, a step is created to list all the assets inside the **assets** directory, which is a directory in your Expo project. The `working_directory` is set to `assets`:
+
+```yaml
+build:
+  name: Demo
+  steps:
+    - run:
+        name: List assets
+        # @info #
+        working_directory: assets
+        # @end #
+        command: ls -la
+```
+
+#### Use a single step
+
+For example, a workflow with the following `steps` will run a single command to install npm dependencies:
+
+```yaml
+build:
+  name: Install npm dependencies
+  steps:
+    # @info #
+    - run:
+        name: Install dependencies
+        command: npm install
+    # @end #
+```
+
+> **info** `-` before `run` counts as indentation.
+
+#### Use multiple steps
+
+When multiple `steps` are defined, they are executed sequentially. For example, a workflow with the following `steps` will first run a single command to install npm dependencies and then run a command to run tests:
+
+```yaml
+build:
+  name: Run tests
+  steps:
+    # @info #
+    - run:
+        name: Install dependencies
+        command: npm install
+    - run:
+        name: Run tests
+        command: |
+          echo "Running tests..."
+          npm test
+    # @end #
+```
+
+## `functions`
+
+Defined to describe a reusable function or [built-in EAS functions](#built-in-eas-functions) that can be used in a workflow. All config options to create a function are specified with the following properties:
+
+### `name`
+
+The name of a function that is used in a `build`. It is also used to identify the function triggered in the build logs. For example, a function named `greetings` is created:
+
+```yaml
+functions:
+  greetings:
+    # @info #
+    name: Say Hi!
+    # @end #
+```
+
+### `platforms`
+
+Name of the platform that a specific function should get executed. `linux` is used for Android and `darwin` for iOS. If not specified, the function is executed for both platforms when the appropriate build runs for each platform.
```

Comment:
It's more of a platform check. The list specifies the platforms that the function is compatible with. Anyway, let's comment this out for now. It's not implemented.